### PR TITLE
Update venue address

### PR DESCRIPTION
**Pull request by Bubblemelon**
Related issue: #175 . See issue :eyes: for screenshot with updates.

### CODE REVIEW

File: 2019/content19.html [hunk 1]
```diff
@@ -109,11 +109,11 @@ <h2>Keynote Speakers</h2>
       <div class="content-box red-box">
         <h2>Venue</h2>
         <p>
-          The main conference talks will be held at the at the University of Maryland, SMC Campus Center, 621 W Lombard St, Baltimore, MD 21201, USA. Here's a map:
+          The main conference talks will be held at the at the University of Maryland, Baltimore SMC Campus Center, 621 W Lombard St, Baltimore, MD 21201, USA. Here's a map:
         </p>
         <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d12352.235663010597!2d-76.6243805!3d39.2869144!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x6adedf5fd2c3d317!2sSMC+Campus+Center!5e0!3m2!1sen!2sro!4v1546879015690" width="100%" height="320" frameborder="0" style="border:0" allowfullscreen></iframe>
         <p>
-          The workshops will be held at the UMB School of Pharmacy nearby.
+          The workshops will be held at the University of Maryland School of Pharmacy nearby.
```
Comment (vjd) [inline 2019/content19.html]: >University of Maryland School of Pharmacy
University of Maryland Baltimore School of Pharmacy

File: 2019/index.html [hunk 2]
```diff
@@ -8,8 +8,8 @@
     <h1 align="center" style="font-weight: 700">JuliaCon is coming to <span style="color: #6581df; font-weight: 1000">Baltimore</span></h1>
     <div class="u-vskip"></div>
     <h2 align="center">
-      <strong>Monday 22nd to Friday 26th of July, 2019</strong> <br/>at the University of Maryland,<br/>
-      <strong>Baltimore, USA</strong>
+      <strong>Monday 22nd to Friday 26th of July, 2019</strong> <br/>at the University of Maryland, Baltimore,<br/>
+      <strong>MD, USA</strong>
```
Comment (vjd) [inline 2019/index.html]: > `<strong>MD, USA</strong>`

`<strong>Baltimore MD, USA</strong>`

File: 2019/content19.html (hunk 1 above)
Comment (vjd) [inline 2019/content19.html]: If you would like to add the address, it is 
20 N Pine St, Baltimore, MD 21201

**Comment by Bubblemelon** [comment]
Ready for review again :smiley: Just for clarity, [this is what the banner looks like](https://github.com/JuliaCon/www.juliacon.org/issues/175#issuecomment-482896095) and [this is for the venue description](https://github.com/JuliaCon/www.juliacon.org/issues/175#issuecomment-483436765).

**Comment by Bubblemelon** [comment]
This PR also takes into account, [PR #178](https://github.com/JuliaCon/www.juliacon.org/pull/178) that was accidentally merged onto `master` instead of `src`.

**Comment by xorJane** [comment]
Thanks @Bubblemelon!

**Pull request merged**